doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -280,12 +280,16 @@ Libraries for networking
   * :ref:`lib_download_client` library:
 
     * Fixed an issue where downloads of COAP URIs would fail when they contained multiple path elements.
+    * Added a parameter :c:member:`set_native_tls` in the configuration structure to configure
+      native TLS support at runtime.
 
   * :ref:`lib_fota_download` library:
 
     * Added:
 
       * :c:func:`fota_download_s0_active_get` function that gets the active B1 slot.
+      * :kconfig:`CONFIG_FOTA_DOWNLOAD_NATIVE_TLS` option for configuring the socket to be native for TLS instead of offloading
+        TLS operations to the modem.
```

Comment:
```suggestion
      * :kconfig:option:`CONFIG_FOTA_DOWNLOAD_NATIVE_TLS` option for configuring the socket to be native for TLS instead of offloading TLS operations to the modem.
```